AOC_Day1/ContentView.swift

```diff
@@ -14,53 +14,43 @@ struct ContentView: View {
     
     
     func rollingTotals(readings: [Int]) -> [Int] {
-        var returnArray: [Int] = []
+        var rollingTotals: [Int] = []
         for i in 0..<readings.count {
-            if i < 2 {
-                continue
-            } else {
-                returnArray.append( readings[i-2] + readings[i-1] + readings[i] )
+            if i >= 2 {
+                rollingTotals.append( readings[i-2] + readings[i-1] + readings[i] )
             }
         }
-        return returnArray
+        return rollingTotals
     }
     
     
     func countIncreases(readings: [Int]) -> Int {
         var counter = 0
         for i in 0..<readings.count {
-            if i == 0 {
-                continue
-            } else {
-                if readings[i] > readings[i-1] {
-                    counter += 1
-                }
+            if i > 0, 
```

Comment:
Again, you can flip the boolean to make this code a little more concise.
Note a cool Swift feature here, you can combine two `if` statements with a comma to meet both conditions.
This will reduce the amount of nesting you need in your code!